dace/transformation/dataflow/map_fission.py

```diff
@@ -195,9 +219,17 @@ def apply(self, graph: sd.SDFGState, sdfg: sd.SDFG):
             for edge in graph.out_edges(map_entry):
                 if edge.data.data:
                     map_syms.update(edge.data.subset.free_symbols)
+                # NOTE: The following code is related to
+                # tests.transformations.mapfission_test.test_mapfission_with_symbols
```

Comment:
strange comment. maybe instead describe the issue and point to the test for reference?